packages/peregrine/lib/ui/components/ProductOptions/swatch.module.css

```diff
@@ -1,21 +1,30 @@
-/* TODO @TW: review. This relies on custom prop to set color from JSX "style" prop. */
-.root {
-    --venia-swatch-bg: var(--venia-global-color-gray-100);
+.base {
+    --venia-swatch-bg: rgb(var(--color-neutral-200));
+    composes: root from '../clickable.module.css';
+
+    composes: rounded-full from global;
+    composes: border-buttonWidth from global;
+    composes: border-solid from global;
+    composes: h-[2.5rem] from global;
+    composes: w-[2.5rem] from global;
```

Comment:
Would the Tailwind basic usage work?

```
composes: h-10 from global;
composes: w-10 from global;
```

Reply:
Can you check the PR for more instances that can use Tailwind's basic usage values? Thank you